client/config-overrides.js

```diff
@@ -0,0 +1,15 @@
+const webpack = require('webpack');
+
+module.exports = function override(config, env) {
+  if (!config.plugins) {
+    config.plugins = [];
+  }
+
+  config.plugins.push(
+    new webpack.DefinePlugin({
+      'process.env.MOCK_USER': JSON.stringify(true),
```

Comment:
ЭЭЭЭ НЕ ВСЕГДА! ты чо... в проде нельзя мокать юзеров ХД

Reply:
@dcversus  ну так на pick-gg-* и так прод стоит, testing не применяется
я 'process.env.NODE_ENV': JSON.stringify(process.env.NODE_ENV) || 'development' добавил, все равно там production

Reply:
так погоди, давай разберемся

где И КОГДА вызывается сборка? - на хероку, на машине сборки
почему и ОТКУДА там берутся env? - задаются в хероку?
ПОЧЕМУ NODE_ENV всегда production? - потому что так захардкожено в базовом react-scripts ты сам так написал
Перестают ли от этого работать ДРУГИЕ env которые задаются в heroku? НЕТ!

Просто бери 
```suggestion
      'process.env.MOCK_USER': JSON.stringify(process.env.MOCK_USER),
```

И отдельно не забудь это на стендах указать)